packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js

```diff
@@ -2537,6 +2537,20 @@ QUnit.module('Visibility of control elements', {
 });
 
 QUnit.module('Events', moduleConfig, () => {
+    QUnit.test('items sizes should be already updated when onResizeEnd is raised (T1240125)', function(assert) {
+        this.reinit({
```

Comment:
Usually, for this kind of test, we should add assert.expect(1) to ensure that onResizeEnd is called after the action. However, we can omit it for this test because there are no other assertions in the test, and the error will be caught here.